packages/xo-server-usage-report/src/index.js

```diff
@@ -703,6 +704,68 @@ const CRON_BY_PERIODICITY = {
   daily: '0 6 * * *',
 }
 
+// let field empty in case of "NaN" and "NONE"
+const CSV_CAST = {
+  number: value => (Number.isNaN(value) ? undefined : String(value)),
+  string: value => (value === 'NONE' ? undefined : value),
```

Comment:
To be coherent with the metric values.

We put an empty field when a value doesn't exists.

Reply:
But why is there a `'NONE'` in the raw data in the first place? Shouldn't it be `undefined` to begin with? (And then the HTML generator would change it to `'NONE'`) And why is the percentage a string? Shouldn't it be a number?

Reply:
It's an old implementation choice, it can be improved. 

I can put `undefined` when there is no evolution in this PR, i just need to ensure that it will not have a side effect on the current plugin behavior.

Reply:
Ok but keeping it as `'NONE'` wouldn't change the current behaviour of the plugin. But if you think it makes more sense to have an empty cell instead, let's keep it like that then.